src/main/java/team/chisel/ctm/client/texture/IMetadataSectionCTM.java

```diff
@@ -63,9 +63,9 @@
             }
         }
         return meta.getType().makeTexture(new TextureInfo(
-                Arrays.stream(ObjectArrays.concat(sprite.getName(), meta.getAdditionalTextures()))
-                	  .map(rl -> new RenderMaterial(AtlasTexture.LOCATION_BLOCKS_TEXTURE, rl))
-                	  .map(bakedTextureGetter::apply)
+                Arrays.stream(ObjectArrays.concat(new ResourceLocation[]{sprite.getName()}, meta.getAdditionalTextures(), ResourceLocation.class))
```

Comment:
This is a change of semantics, as per documentation of the `ObjectArrays#concat` methods

https://guava.dev/releases/21.0/api/docs/com/google/common/collect/ObjectArrays.html#concat-T-T:A-
https://guava.dev/releases/21.0/api/docs/com/google/common/collect/ObjectArrays.html#concat-T:A-T-

Reply:
yes it was, but it didn't want to compile for me, so I "fixed" it. I tried the previous ObjectArrays#concat call and now it works...